src/components/NcRichText/highlight.scss

```
@@ -0,0 +1,195 @@
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+/**
+* Colors and class selectors are extracted from source code of:
+* - library: highlight.js (v11.10.0)
+* - light theme: highlight.js/styles/github.css
+* - dark theme: highlight.js/styles/github-dark.css
+* and reworked to use with Nextcloud dark and light theme
+*/
+
+@mixin highlight-light-colors {
+	--hljs-color: #24292e;
+	--hljs-background-color: #ffffff;
```

I think this should always be background dark, so we are consistent with text etc.

```suggestion
	--hljs-background-color: var(--color-background-dark);
```

---

I kept original colors as a fallback (see L83:L84)
Though it's never be used, could be also set directly here

---

Ah yes saw that now, so I am fine with both